src/test/java/com/serenitydojo/calculator/WhenDoingMaths.java

```diff
@@ -13,38 +13,38 @@ public void shouldReturnZeroForAnEmptyString() {
         assertThat(calculator.evaluate("")).isEqualTo(0);
     }
 
-//    @Test(expected = IllegalMathsOperatorException.class)
-//    public void shouldReportNonSupportedOperations() {
-//        calculator.evaluate("1 ^ 2");
-//    }
-//
-//    @Test
-//    public void shouldReturnTheValueOfASingleNumber() {
-//        assertThat(calculator.evaluate("1")).isEqualTo(1);
-//    }
-//
-//    @Test
-//    public void shouldAddTwoNumbers() {
-//        assertThat(calculator.evaluate("1 + 1")).isEqualTo(2);
-//    }
-//
-//    @Test
-//    public void shouldAddThreeNumbers() {
-//        assertThat(calculator.evaluate("1 + 2 + 3")).isEqualTo(6);
-//    }
-//
-//    @Test
-//    public void shouldAlsoSubtract() {
-//        assertThat(calculator.evaluate("10 - 6")).isEqualTo(4);
-//    }
-//
-//    @Test
-//    public void shouldAddAndSubtract() {
-//        assertThat(calculator.evaluate("10 + 5 - 6")).isEqualTo(9);
-//    }
-//
-//    @Test
-//    public void shouldMultiplyNumbers() {
-//        assertThat(calculator.evaluate("10 * 5")).isEqualTo(50);
-//    }
+    @Test(expected = IllegalMathsOperatorException.class)
+    public void shouldReportNonSupportedOperations() {
+        calculator.evaluate("1 ^ 2");
+    }
+
+    @Test
+    public void shouldReturnTheValueOfASingleNumber() {
+        assertThat(calculator.evaluate("1")).isEqualTo(1);
+    }
+
+    @Test
+    public void shouldAddTwoNumbers() {
+        assertThat(calculator.evaluate("1 + 1")).isEqualTo(2);
+    }
+
+    @Test
+    public void shouldAddThreeNumbers() {
+        assertThat(calculator.evaluate("1 + 2 + 3")).isEqualTo(6);
+    }
+
+    @Test
+    public void shouldAlsoSubtract() {
+        assertThat(calculator.evaluate("10 - 6")).isEqualTo(4);
+    }
+
+    @Test
+    public void shouldAddAndSubtract() {
+        assertThat(calculator.evaluate("10 + 5 - 6")).isEqualTo(9);
+    }
+
+    @Test
+    public void shouldMultiplyNumbers() {
+        assertThat(calculator.evaluate("10 * 5")).isEqualTo(50);
+    }
```

Comment:
This is all great stuff - nice clear assertions.

Reply:
Thx a lot John, but I can't take the compliment : the assertions where part of the initial state of the project, I just uncommented them